sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/test/environment/setup/AzureSearchResources.java

```diff
@@ -121,7 +122,7 @@ public void createResourceGroup() {
         String resourceGroupName = Configuration.getGlobalConfiguration().get(AZURE_RESOURCEGROUP_NAME);
```

Comment:
Since there is a default hardcoded test resource group you can get rid of the null or empty check and just use the following here.

```java
String resourcegroupname = Configuration.getGlobalConfiguration().get(AZURE_RESOURCEGROUP_NAME, TEST_RESOURCE_GROUP);
```

This is effectively a get or default.